plugins/kubernetes-backend/src/cluster-locator/ConfigClusterLocator.ts

```diff
@@ -50,6 +51,9 @@ export class ConfigClusterLocator implements KubernetesClustersSupplier {
         if (c.has('dashboardParameters')) {
           clusterDetails.dashboardParameters = c.get('dashboardParameters');
         }
+        if (c.has('displayName')) {
+          clusterDetails.displayName = c.get('displayName');
```

Comment:
I think `getOptionalString` fits best here right?

Reply:
Good catch, thank you! Fixed.